apps/prairielearn/src/schemas/schemas/infoQuestion.json

```diff
@@ -57,23 +58,28 @@
     },
     "gradingMethod": {
       "description": "The grading method used for this question.",
+      "type": "string",
       "enum": ["Internal", "External", "Manual"]
     },
     "singleVariant": {
       "description": "Whether the question is not randomized and only generates a single variant (defaults to \"false\").",
-      "type": "boolean"
+      "type": "boolean",
+      "default": false
     },
     "showCorrectAnswer": {
       "description": "Whether to show the correct answer panel (defaults to \"true\").",
-      "type": "boolean"
+      "type": "boolean",
+      "default": true
```

Comment:
Nit: with the default here, should we remove the "defaults ..." from the description above? Ditto elsewhere.

Reply:
https://github.com/PrairieLearn/PrairieLearn/issues/11256 getting some deja vu 😄

Reply:
Ha, nice! At least I'm consistent with my comments over time :joy: I went ahead and closed that issue, it's resolved now.